en/flight_modes/flight_modes_summary.md

```diff
@@ -191,93 +181,126 @@ th {
 <tbody>
 
 <tr>
- <td>Manual/ Stabilized</td>
+ <td><a id="manual_stabilized_mc" href="../flight_modes/manual_stabilized_mc.md">Manual/ Stabilized</a></td>
  <td>S</td>
  <td>S<sub>rate</sub></td>
  <td>M</td>
  <td></td>
- <td><p>This mode allows manual control with assistance from autopilot to stabilize attitude.</p>
-<p>Centered RC sticks level-out the attitude.</p>
+ <td><p>RC/manual mode where centered sticks level vehicle (position is not stabilized).</p>
+   <p>
+   <ul>
+    <li>Centered RP sticks level vehicle.</li>
+    <li>Outside center:
+      <ul>
+       <li>RP sticks control left/right and forward/back speed (respectively).</li>
```

Comment:
I'm not sure about this. The stick controls the tilt angle, which then affects the vehicle's speed. To control the speed directly, you would need to have position/velocity information.

Reply:
That was intentional - the user doesn't really care about angles, just the way the vehicle moves (ie left /right). But I thought about this a bit and this is more a "techical" version, so now try say both (added inhttps://github.com/PX4/px4_user_guide/pull/185)

Basically it now says:


```
       <li>Roll/Pitch sticks control tilt angle in those orientations, resulting in corresponding left-right and forward-back movement.</li>
       <li>Throttle stick controls up/down speed (and movement speed in other axes).</li>
       <li>Yaw stick controls rate of angular rotation above the horizontal plane.</li>
```

Reply:
Thanks